### PR TITLE
Check glide version

### PR DESCRIPTION
**Pull request by soltysh**
Same thing as in openshift/client-go#19

@deads2k ptal

### CODE REVIEW

**Comment by sttts** [comment]
Same applies as for the client-go counterpart.

**Comment by soltysh** [comment]
Nits updated.

File: hack/update-deps.sh [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+
+function verify_glide_version() {
+	if ! command -v glide &> /dev/null; then
+		echo "[FATAL] Glide was not found in \$PATH. Please install version ${REQUIRED_GLIDE_VERSION} or newer."
+		exit 1
+	fi
+
+	local glide_version
+	glide_version=($(glide --version))
+	if [[ "${glide_version[2]}" < "${REQUIRED_GLIDE_VERSION}" ]]; then
```
Comment (sttts) [inline hack/update-deps.sh]: REQUIRED_GLIDE_VERSION has a `v` as first character. My glide has not. This feels broken.

Comment (soltysh) [reply hack/update-deps.sh]: That should be easily fixable...

Comment (soltysh) [reply hack/update-deps.sh]: It wasn't not that easy, but it's doable 😉

**Comment by soltysh** [comment]
@sttts updated

**Comment by sttts** [comment]
/lgtm

Don't let any bash perfectionist look at it.

**Pull request merged**